containers/ecr-viewer/src/app/admin/user/UserTable.tsx

```diff
@@ -198,6 +259,69 @@ const FilterByUserType = ({
   );
 };
 
+const FilterByProgramArea = ({
+  isAllSelected,
+  filterProgramAreas,
+  setFilterProgramAreas,
+}: {
+  isAllSelected: boolean;
+  filterProgramAreas: FilterProgramAreasType;
+  setFilterProgramAreas: Dispatch<SetStateAction<FilterProgramAreasType>>;
+}) => {
+  const handleSelectDeselectAll = (
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    const { checked } = event.target;
+    const updatedProgramAreas = Object.keys(filterProgramAreas).reduce(
+      (acc, programName) => {
+        acc[programName] = checked;
+        return acc;
+      },
+      {} as FilterProgramAreasType,
+    );
```

Comment:
if we change the event handler to `() => void`
```suggestion
  const handleSelectDeselectAll = (
  ) => {
    const updatedProgramAreas = Object.keys(filterProgramAreas).reduce(
      (acc, programName) => {
        acc[programName] = !isAllSelected;
        return acc;
      },
      {} as FilterProgramAreasType,
    );
```

Reply:
@mcmcgrath13 👍  Also would have to make a similar change in `EcrFilters` changing `handleSelectAll` to a  `() => void`